test/manipulation.test.js

```diff
@@ -688,6 +688,313 @@ describe('manipulation', function () {
     });
   });
 
+  if (!getSchema().connector.replaceOrCreate) {
+    describe.skip('replaceOrCreate - not implemented', function(done) { done; });
```

Comment:
So we don't want to have tests for other databases for which we use "non-atomic" database query?